notifiers/slack.go

```diff
@@ -32,7 +33,7 @@ func (n *slackNotifier) Send(notification Notification, recipient string) error
 		},
 	}
 	s := slack.New(n.opts.Token, slack.OptionHTTPClient(client))
-	msgOptions := []slack.MsgOption{slack.MsgOptionText(notification.Body, false)}
+	msgOptions := []slack.MsgOption{slack.MsgOptionText(notification.Body, false), slack.MsgOptionUsername(n.opts.Username)}
```

Comment:
What happens if username is empty? I think we need to add a check and use `slack.MsgOptionUsername` only if `Username` is not empty string.

Reply:
If username is empty, the username set in token takes precedence.
However, it is difficult to understand, so we will add a check.

Reply:
thank you.